ci/deploy-gh-pages.sh

```diff
@@ -1,21 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Deploy repository documentation
 
 # Variables
-bold=$(tput bold)
-normal=$(tput sgr0)
-script_path="$( cd "$(dirname "$0")" ; pwd -P )"
-project_path=${script_path}/..
+doc_path="doc"
+project_path=${doc_path}/..
 clone_folder=/tmp/update-repos
 
 repository_name="ping-python"
 repository_githash=$(git -C ${project_path} rev-parse HEAD)
 
 # Functions
-echob() {
-    echo "${bold}${1}${normal}"
-}
+. ci/ci-functions.sh
```

Comment:
moving to local folder is not necessary, use `./`

Reply:
This is not executing, this is [_sourcing_](https://superuser.com/questions/176783/what-is-the-difference-between-executing-a-bash-script-vs-sourcing-it).

Reply:
So why not `source` ?

Reply:
They are synonomous: 
![image](https://user-images.githubusercontent.com/8315108/60603214-5262e400-9d83-11e9-8dc2-771468858089.png)

I can change it.